string      cluster_id   # Identifier for the model to which the inliers belong to.
int32[]     inliers      # Array of inliers indices